sdk/identity/identity/src/credentials/credentialPersistenceOptions.ts

```diff
@@ -12,21 +12,21 @@ export interface CredentialPersistenceOptions {
    * Options to provide to the persistence layer (if one is available) when
    * storing credentials.
    * 
-   * You must first register a persistence provider as an extension. See the
+   * You must first register a persistence provider plugin. See the
    * `@azure/identity-cache-persistence` package on NPM.
    * 
    * Example:
    * 
    * ```typescript
-   * import persistence from "@azure/identity-cache-persistence";
-   * import { useIdentityExtension, DeviceCodeCredential } from "@azure/identity";
+   * import { cachePersistencePlugin } from "@azure/identity-cache-persistence";
+   * import { useIdentityPlugin, DeviceCodeCredential } from "@azure/identity";
    * 
-   * useIdentityExtension(persistence);
+   * useIdentityPlugin(cachePersistencePlugin);
    * 
    * async function main() {
    *   const credential = new DeviceCodeCredential({
    *     tokenCachePersistenceOptions: {
-   *       name: "mycustomcachename"
```

Comment:
wait, is this change intended?

Reply:
Yes, the cache name is not required and probably shouldn’t be set ordinarily (as far as I understand it), so I think it makes sense to show just the required `enabled` property in the example. The full documentation will of course still show the `name` field as an option.

Reply:
Alright! thank you!